docs/settings.md

```diff
@@ -0,0 +1,55 @@
+# Settings
+
+Copier settings are stored in `<CONFIG_ROOT>/settings.yml` where `<CONFIG_ROOT>` is the
+stdandrd configuration directory for your platform:
+
+-   `$XDG_CONFIG_HOME/copier` (`~/.config/copier ` in most cases) on Linux as defined by
+    [XDG Base Directory Specifications](https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html)
+-   `~/Library/Application Support/copier` on macOS as defined by
+    [Apple File System Basics](https://developer.apple.com/library/archive/documentation/FileManagement/Conceptual/FileSystemProgrammingGuide/FileSystemOverview/FileSystemOverview.html)
+-   `%USERPROFILE%\AppData\Local\copier` on Windows as defined in
+    [Known folders](https://docs.microsoft.com/en-us/windows/win32/shell/known-folders)
+
+This location can be overridden by setting the `COPIER_SETTINGS` environment variable.
+
+## User defaults
+
+User may define some reusable defaults variables in the `defaults` section of the
+configuration file.
+
+```yaml title="$COPIER_HOME/settings.yml"
+defaults:
+    user_name: "John Doe"
+    user_email: john.doe@acme.com
+```
+
+Those user data will replace the default value of fieds of the same name.
+
+### Well known variables
+
+To ensure templates efficiently reuse user defined variables, we invite template authors
+to use the following well known variables:
+
+| variable name | type  | description            |
```

Comment:
I think it's nicer to capitalize the table header:

```suggestion
| Variable name | Type  | Description            |
```

Reply:
I'll also note that the wrapping pipes (left-most and right-most ones) are not mandatory :slightly_smiling_face: